### DISCUSSION TITLE
Decoder (Transformers) – ForBo7 // Salman Naqvi

### DISCUSSION THREAD

**Discussion by giscus** [ForBo7/forbo7.github.io#13] [Giscus Comments]
# Decoder (Transformers) – ForBo7 // Salman Naqvi

A compoent of a trasnformer that is used for generating text. An example is the autocomplete feature on a smartphone’s keyboard.

https://forbo7.github.io/dictionary/terms/decoder.html

**Comment by slf188** [comment]
component

**Reply by slf188** [reply]
transformer

**Reply by ForBo7** [reply]
I need to start paying closer attention 👀
